niceman/distributions/tests/test_conda.py

```diff
@@ -0,0 +1,64 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil; coding: utf-8 -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the niceman package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+import collections
+import os
+
+import sys
+from subprocess import call
+
+import yaml
+import attr
+from niceman.formats.niceman import NicemanProvenance
+
+try:
+    import apt
+except ImportError:
+    apt = None
+
+import json
+
+from niceman.distributions.conda import CondaTracer
+
+
+def create_test_conda():
+    if os.path.exists("/tmp/niceman_conda_test"):
+        return
+    call("mkdir /tmp/niceman_conda_test; "
+         "cd /tmp/niceman_conda_test; "
+         "curl -O https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh; "
+         "bash -b Miniconda2-latest-Linux-x86_64.sh -b -p /tmp/niceman_conda_test/miniconda; "
+         "/tmp/niceman_conda_test/miniconda/bin/conda create -y -n mytest python=2.7;"
+         "/tmp/niceman_conda_test/miniconda/envs/mytest/bin/conda install -y xz -n mytest;"
+         "/tmp/niceman_conda_test/miniconda/envs/mytest/bin/pip install pluggy;",
+         shell=True)
```

Comment:
use random name directory (wouldn't persist across runs) or just use "standard" cache directory for user-level niceman configuration

```ipython
In [1]: from appdirs import AppDirs

In [2]: dirs = AppDirs('niceman')

In [3]: dirs.user_cache_dir
Out[3]: '/home/yoh/.cache/niceman'

In [4]: os.path.join(dirs.user_cache_dir, 'tests', 'installations') 
```

Reply:
Done.  Thank you!